vmngclient/endpoints/configuration/software_actions.py

```diff
@@ -0,0 +1,187 @@
+# mypy: disable-error-code="empty-body"
+
+from enum import Enum
+from typing import List, Optional, Union
+from uuid import UUID
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from vmngclient.endpoints import APIEndpoints, delete, get, post, put
+from vmngclient.typed_list import DataSequence
+
+
+class ImageType(str, Enum):
```

Comment:
It might be a good idea to move models from `endpoints` directory to `models` directory. At least for these for which is clear that they could be reused or not firmly attached to single endpoint.

Reply:
Will do when replacing all endpoints in software upgrades API